californium-core/src/main/java/org/eclipse/californium/core/network/config/NetworkConfigDefaults.java

```diff
@@ -76,7 +76,7 @@ public static void setDefaults(final NetworkConfig config) {
 		config.setInt(NetworkConfig.Keys.PREFERRED_BLOCK_SIZE, 512);
 		config.setInt(NetworkConfig.Keys.MAX_MESSAGE_SIZE, 1024);
 		config.setInt(NetworkConfig.Keys.MAX_RESOURCE_BODY_SIZE, DEFAULT_MAX_RESOURCE_BODY_SIZE);
-		config.setInt(NetworkConfig.Keys.BLOCKWISE_STATUS_LIFETIME, 10 * 60 * 1000); // ms
+		config.setInt(NetworkConfig.Keys.BLOCKWISE_STATUS_LIFETIME, DEFAULT_BLOCKWISE_STATUS_LIFETIME); // ms
```

Comment:
This is the maximum time span *between* subsequent requests of the blockwise transfer. The timer is reset every time we receive a new request for an inbound block1 or block2 request.
I reduced this value drastically because I wasn't aware that it actually needs to be very high in order to support low-bandwidth, low-reliability transfers.
I have no problem with increasing it again. The Blockwise Transfer RFC mandates that clients do all required block transfers in rapid succession without further delays in between. I therefore assume that a value based on  EXCHANGE_LIFETIME could serve well.